site/content/docs/1.25/guides/external-authorization.md

```diff
@@ -414,7 +414,7 @@ date: Mon, 20 Feb 2023 13:45:31 GMT
 If you add the username and password to the same request you can verify that the request succeeds. 
 ```
 $ curl -k --user user1:password1 http://local.projectcontour.io/test/$((RANDOM))
-{"TestId":"","Path":"/test/27748","Host":"local.projectcontour.io","Method":"GET","Proto":"HTTP/1.1","Headers":{"Accept":["*/*"],"Auth-Context-Header1":["value1"],"Auth-Context-Header2":["value2"],"Auth-Context-Routq":["global"],"Auth-Handler":["htpasswd"],"Auth-Realm":["default"],"Auth-Username":["user1"],"Authorization":["Basic dXNlcjE6cGFzc3dvcmQx"],"User-Agent":["curl/7.86.0"],"X-Envoy-Expected-Rq-Timeout-Ms":["15000"],"X-Envoy-Internal":["true"],"X-Forwarded-For":["172.18.0.1"],"X-Forwarded-Proto":["http"],"X-Request-Id":["b6bb7036-8408-4b03-9ce5-7011d89799b4"],"X-Request-Start":["t=1676900780.118"]}}
+{"TestId":"","Path":"/test/27748","Host":"local.projectcontour.io","Method":"GET","Proto":"HTTP/1.1","Headers":{"Accept":["*/*"],"Auth-Context-Header1":["value1"],"Auth-Context-Header2":["value2"],"Auth-Context-Route":["global"],"Auth-Handler":["htpasswd"],"Auth-Realm":["default"],"Auth-Username":["user1"],"Authorization":["Basic dXNlcjE6cGFzc3dvcmQx"],"User-Agent":["curl/7.86.0"],"X-Envoy-Expected-Rq-Timeout-Ms":["15000"],"X-Envoy-Internal":["true"],"X-Forwarded-For":["172.18.0.1"],"X-Forwarded-Proto":["http"],"X-Request-Id":["b6bb7036-8408-4b03-9ce5-7011d89799b4"],"X-Request-Start":["t=1676900780.118"]}}
```

Comment:
This change:

```diff
-Auth-Context-Routq
+Auth-Context-Route
```

is perhaps notable. One might assume that this output was generated by a machine, and thus I'm concerned that any machiine would generate `Routq`. It certainly wasn't a typo I fixed in the code (it's only present in the documentation from 1.25 through main).

I do try to ensure that programmatic output matches what documentation says. -- It's improper to "fix" documentation in a way that desyncs it from reality.